wye/templates/workshops/workshop_detail.html

```diff
@@ -9,14 +10,33 @@ <h1 class="no-space">Workshop Details</h1>
     <div class="push-4-bottom">
     	{% if user.is_authenticated %}
         <div class="row push-1-bottom">
-            <div class="col-xs-6 sub">
+            <div class="col-xs-5 sub">
                 <a href="{% url 'workshops:workshop_list' %}"><i class="fa fa-arrow-left"></i>&nbsp;&nbsp;My Workshops</a>
             </div>
+            <div class="col-xs-7 text-right">
             {% if display_edit_button %}
-            <div class="col-xs-6 text-right">
                 <a href="{% url 'workshops:workshop_update' workshop.pk %}" class="btn btn-theme-primary sub pad-1"><i class="fa fa-cog"></i>&nbsp;&nbsp;Edit<span class="hidden-xs"> Workshop</span></a>
-            </div>
+                {% if workshop|show_draft_button:user  %}
```

Comment:
Sorry I missed this comment. The check is already there for the whole row (see line 11)